### ISSUE TITLE
LocalContentUriFetchProducer "contacts/42/photo" not loading

### ISSUE THREAD

**Issue by chonamdoo** [facebook/fresco#88]
can't open '/contacts/42/photo'

**Comment by tyronen** [comment]
Does your app have this line in its `AndroidManifest.xml` file?

```xml
<uses-permission android:name="android.permission.READ_CONTACTS" />
```

**Comment by chonamdoo** [comment]
yes  glide  open T T

**Comment by chonamdoo** [comment]
yes AndroidManifest.xml
already <uses-permission android:name="android.permission.READ_CONTACTS"/>
why not open
2015. 4. 4. 오전 2:46에 "Tyrone Nicholas" <notifications@github.com>님이 작성:

**Comment by chonamdoo** [comment]
hmm TT not open

**Comment by raymoondtang** [comment]
Since the duplicated issue #97 is closed. I will copy my comment here. 
It's an issue with how LocalContentUriFetchProducer get the input stream from Uri.
A workaround is to remove '/photo' at the end of URI, it will load, but not for all devices as I tested.

**Comment by facebook-github-bot-0** [comment]
Thank you for reporting this issue and appreciate your patience. We've  notified the core team for an update on this issue. We're looking for a response within the next 30 days or the issue may be closed.

**Comment by tyronen** [comment]
Fixed in v0.8.0.
